### PR TITLE
W-11611396 revamped to avoid Roles and redundancy

### PR DESCRIPTION
**Pull request by hannanelson**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/mq-access-management.adoc [hunk 1]
```diff
@@ -11,97 +11,65 @@ Anypoint Platform enables you to manage the roles of users who administer the An
...
 +
+** Create new queues, message exchanges, and bindings
+** Edit existing queues, message exchanges, and bindings
+** Delete messages
+Manage destinations (deprecated)::
 Same privileges as *View destinations* and *Clear destinations*, plus permission to:
++
 ** Create new queues and message exchanges
 ** Edit existing queues and message exchange settings
-** Access *Message Sender* and *Browser* pages
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-access-management.adoc:73]: Manage Destination role still has these permissions. Can you add this line back `Access *Message Sender* and *Browser* pages` under Manage Destination

Comment (hannanelson) [reply modules/ROOT/pages/mq-access-management.adoc:73]: done

File: modules/ROOT/pages/mq-access-management.adoc [hunk 2]
```diff
@@ -11,97 +11,66 @@ Anypoint Platform enables you to manage the roles of users who administer the An
...
-+
+Manage clients::
 Same privileges as *View clients*, plus permission to create client apps
-* *Manage destinations*
+Administer destinations::
+Same privileges as *View destinations* and *Clear destinations*, plus permission to:
 +
+** Create new queues, message exchanges, and bindings
+** Edit existing queues, message exchanges, and bindings
+** Delete messages
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-access-management.adoc]: We need to remove delete messages from `Administer destinations` permissions

File: modules/ROOT/pages/mq-access-management.adoc [hunk 3]
```diff
@@ -11,97 +11,66 @@ Anypoint Platform enables you to manage the roles of users who administer the An
...
-+
-You can click *Select all* to set all permissions, or *Select none* to erase previously clicked settings.
-. Click the blue plus icon (+) to add the selection.
-+
-The selected environment and permissions appear below the drop-down list.
 +
-You can delete items from this list by clicking the delete icon.
+[NOTE]
+This permission is deprecated; use *Administer destinations* instead.
+Destination subscriber for given environment::
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-access-management.adoc:75]: Add `delete messages` under `subscribe` role

File: modules/ROOT/pages/mq-access-management.adoc [hunk 4]
```diff
@@ -11,97 +11,66 @@ Anypoint Platform enables you to manage the roles of users who administer the An
...
+Manage clients::
 Same privileges as *View clients*, plus permission to create client apps
-* *Manage destinations*
+Administer destinations::
+Same privileges as *View destinations* and *Clear destinations*, plus permission to:
 +
+** Create new queues, message exchanges, and bindings
+** Edit existing queues, message exchanges, and bindings
+** Delete messages
+Manage destinations (deprecated)::
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-access-management.adoc:58]: Can we somehow add that, if we want same set of permissions as `manage destination` role ,  we can add  Administer destinations to user along with `Destination subscriber for given environment` and `Destination publisher for given environment`

File: modules/ROOT/pages/mq-access-management.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-access-management.adoc]: done

File: modules/ROOT/pages/mq-access-management.adoc (hunk 4 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-access-management.adoc:58]: done

File: modules/ROOT/pages/mq-access-management.adoc (hunk 3 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-access-management.adoc:75]: done

**Review by kimayap03 — DISMISSED** [review]
LGTM!

File: modules/ROOT/pages/mq-access-management.adoc [hunk 5]
```diff
@@ -11,97 +11,74 @@ Anypoint Platform enables you to manage the roles of users who administer the An
...
+== Anypoint MQ Permissions
 
-* *Clear destinations*
-+
+The Anypoint MQ permissions are:
+
+View clients::
+View all client apps, as well as the client app IDs and client secrets for each client app
+View destinations::
+View all destinations and each destination's settings (*ID*, *Type*, *Message TTL*, and *Message Lock Default TTL*). View *In Queue* messages and *In Flight* message stats.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/mq-access-management.adoc]:
```suggestion
View all destinations and each destination's settings (*ID*, *Type*, *Message TTL*, and *Message Lock Default TTL*), and view *In Queue* messages and *In Flight* message stats.
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/mq-access-management.adoc]: just trying to get rid of the period since it's the only item in the list with one

**Review by IsaacEldridge — DISMISSED** [review]

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**